mtp_noms_ops/templates/security/accept_or_reject_check.html

```diff
@@ -177,13 +181,53 @@ <h3 class="heading-medium">{% trans 'Credit decisions matching debit card or pri
       {% if check.status == 'pending' %}
 
         <div class="form-group">
-          <h2 class="heading-medium">{% trans 'Your decision' %}</h2>
-          <div class="form-group">
-            {% include 'mtp_common/forms/textarea.html' with field=form.decision_reason input_classes='form-control-3-4' %}
+          <h2 class="heading-medium">{% trans 'Accept or reject this credit' %}</h2>
+          <h3 class="heading-medium">{% trans 'To accept this credit:' %}</h2>
+          <div>
+            {% include 'mtp_common/forms/textarea.html' with field=form.accept_further_details input_classes='form-control-3-4' %}
             <button type="submit" class="button" name="fiu_action" value="accept">
               {% trans 'Accept credit' %}
             </button>
-            <button type="submit" class="button button-warning" name="fiu_action" value="reject">
+          </div>
+
+          <h3 class="heading-medium">{% trans 'To reject this credit:' %}</h2>
+          <fieldset class="mtp-check-form-spacing">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend">{% trans 'Select all reasons for rejecting this credit:' %}</legend>
+            {% include 'mtp_common/forms/checkbox-field.html' with field=form.payment_source_paying_multiple_prisoners input_classes='form-control-3-4' %}
+            {% include 'mtp_common/forms/checkbox-field.html' with field=form.payment_source_multiple_cards input_classes='form-control-3-4'  %}
+            {% include 'mtp_common/forms/checkbox-field.html' with field=form.payment_source_linked_other_prisoners input_classes='form-control-3-4'  %}
+            {% include 'mtp_common/forms/checkbox-field.html' with field=form.payment_source_known_email input_classes='form-control-3-4' %}
+            {% include 'mtp_common/forms/checkbox-field.html' with field=form.payment_source_unidentified input_classes='form-control-3-4' %}
+            {% include 'mtp_common/forms/checkbox-field.html' with field=form.prisoner_multiple_payments_payment_sources input_classes='form-control-3-4' %}
+            <div class="multiple-choice form-group" data-target="fiu_investigation_id">
```

Comment:
actually, there is a minor point. it's minor cos these classes will change "soon" anyway as part of the design system update.

i think these all should just be `multiple-choice`, not `multiple-choice form-group`. when they open up, they look a bit too tightly positioned. [gds](https://govuk-elements.herokuapp.com/form-elements/#form-toggle-content) don't put form-group here. i only did the briefest of checks in chrome dev tools, so there might be some other issues/side effects to your new js toggle etc so don't just trust me on this one!

Reply:
The uniform spacing of the check-toggled text fields with respect to the other fields seems to match justin's design. https://pm-intelligence-tool.herokuapp.com/delayed_capture_pause_03/review_0 It calls for applying uniform styling via the outer `<div>` to these check-toggled text fields when in their collapsed state and the fields being templated via `checkbox-field.html`. Without this styling the check-toggled text fields are much more distantly spaced, and it looks a bit naff. 

The reason for not using two different styling classes (i.e. a separate class for these check-toggled text fields) is that I ran into a weird issue where the margins of these fields don't stack properly, so I had to override the behaviour of all `div` elements with `form-group` class that are child elements of the `mtp-check-form-spacing` class to make their `margin-bottom` zero-width to remove all stacking in order to get these elements consistent. Currently there's no way to add a custom class to the outer `div` of the `checkbox-field.html` currently, and I was reticent to change the templating to accommodate this (even though this template snippet doesn't seem to be used anywhere else that I can find). I could add a similar rule that targets all `div` elements with `multiple-class` class that are child elements of the `mtp-check-form-spacing` class, but it felt like unnecessary duplication.